### PR TITLE
chore: fix ci tests again

### PR DESCRIPTION
**Pull request by xiaoshihou514**
a commit changed clang-format's default so the :extra test fails

### CODE REVIEW

File: test/filetype_spec.lua [hunk 1]
```diff
@@ -156,7 +156,7 @@ describe('filetype module', function()
     ft('c'):fmt('clang-format'):extra('--verbose'):lint('clang-tidy'):extra('--fix')
     same({
       cmd = 'clang-format',
-      args = { '--verbose', '-style=file' },
```
Comment (glepnir) [inline test/filetype_spec.lua:159]: hmm nope this should keep . it should insert extra arguments  to default args

**Comment by glepnir** [comment]
looks like #43 remove `--stlye=file`.

File: test/filetype_spec.lua (hunk 1 above)
Comment (xiaoshihou514) [reply test/filetype_spec.lua:159]: ok

**Review by glepnir — APPROVED** [review]

File: test/filetype_spec.lua (hunk 1 above)
Comment (barrett-ruth) [reply test/filetype_spec.lua:159]: Clang format default uses that style BTW. It says in --help

**Pull request merged**